scripts/localstorage.js

```diff
@@ -0,0 +1,15 @@
+const contactForm1 = document.querySelector('.contactForm');
+contactForm1.addEventListener('input', () => {
+  const formInfo = {
+    name: document.querySelector('.contactName').value,
+    email: document.querySelector('.emailAddress').value,
+    message: document.querySelector('.contactMessage').value,
+  };
+
+  localStorage.setItem('contactFormContent', JSON.stringify(formInfo));
+});
+
+const formObj = JSON.parse(localStorage.getItem('contactFormContent'));
+document.querySelector('.contactName').value = formObj.name;
+document.querySelector('.emailAddress').value = formObj.email;
+document.querySelector('.contactMessage').value = formObj.message;
```

Comment:
- [ ] Good job so far 👍 but your data is not being preserved in the browser. Kindly fix this issue and verify that the form data is gotten upon refreshing or reloading the page